API/Controllers/ProjectController.cs

```diff
@@ -239,6 +239,35 @@ public async Task<IActionResult> MigrateDatabaseToElasticSearch()
             return Ok(resultsResource);
         }
 
+        /// <summary>
+        ///     This method returns suggestions while searching for projects
+        /// </summary>
+        /// <param name="query"></param>
+        /// <returns>This method returns a list of autocomplete project resources.</returns>
+        /// <response code="200">This endpoint returns a list with suggested projects.</response>
+        [HttpGet("search/autocomplete")]
+        [ProducesResponseType(typeof(List<AutocompleteProjectResourceResult>), (int) HttpStatusCode.OK)]
+        public async Task<IActionResult> GetAutoCompleteProjects([FromQuery(Name ="query")] string query)
+        {
+            try
+            {
+                List<Project> projects = await projectService.FindProjectsWhereTitleStartsWithQuery(query);
+                List<AutocompleteProjectResourceResult> autocompleteProjectResourceResults = mapper.Map<List<Project>, List<AutocompleteProjectResourceResult>>(projects);
+                return Ok(autocompleteProjectResourceResults);
+            }
+            catch(Exception)
```

Comment:
I don't like to catch on the base Exception. I think whenever an unexpected exception occurs, we do not want to catch this, only the 'expected exception'. Maybe we can reuse an existing exception or create a new exception (an UnavailableServiceException?).